app/views/invite_requests/status.html.erb

```diff
@@ -0,0 +1,26 @@
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading">
+  <%= ts("Invitation Request Status") %>
+</h2>
+
+<p>
+  <%= ts("There are currently %{count} people on the waiting list.", count: InviteRequest.count) %>
+  <% if @admin_settings.invite_from_queue_enabled? %>
+    <%= ts("We are sending out %{invites} invitations per day.", invites: AdminSetting.invite_from_queue_number) %>
```

Comment:
Should we use `@admin_settings` for caching, as in [AO3-4470](https://otwarchive.atlassian.net/browse/AO3-4470)?